### ISSUE TITLE
windows compatible?

### ISSUE THREAD

**Issue by abdelhakim96** [kul-optec/alpaqa#13]
Is alpaca windows compatible?

**Comment by tttapa** [comment]
Yes, alpaqa is compatible with Windows, macOS and Linux. You can install it (on all three platforms) using `python3 -m pip install --upgrade --pre alpaqa`.

To be able to pre-compile CasADi problems, you'll need to have a C compiler installed (e.g. Visual Studio), see https://kul-optec.github.io/alpaqa/Sphinx/usage/tips-and-tricks.html#compilation and https://kul-optec.github.io/alpaqa/Sphinx/usage/tips-and-tricks.html#compiler-installation for some more details.

**Comment by abdelhakim96** [comment]
Thank you so much! Just another question, I saw there is a conan file, is it conan 2 recipe? Also is there a plan to add it to conan center? Thanks

**Comment by tttapa** [comment]
Yes, it's a Conan 2 recipe. You can find some instructions here: https://github.com/kul-optec/alpaqa/tree/develop/examples/CMake/Solver#readme

I plan to add it to Conan Center once I have the 1.0 release ready.

**Comment by abdelhakim96** [comment]
Awesome!! Thank you so much
